templates/travis/.travis/script.sh.j2

```diff
@@ -28,6 +28,10 @@ if [ "$TEST" = "docs" ]; then
   exit
 fi
 
+if [[ "$TEST" == "plugin-from-pypi" ]]; then
+  git checkout {{ stable_tag }}
```

Comment:
This looks dangerous, as it changes all the files you are already working with in flight.
Maybe you can restrict the checkout to the `tests` directory.